src/lib.rs

```diff
@@ -99,7 +98,9 @@ impl UserdataGuard {
                     },
                 )
             })
-            .unwrap_or_else(|_: AccessError| Err(UserdataError::AccessError))
```

Comment:
This is the one place I wasn't very sure that the new version behaves the same way as the old version

Reply:
Yep, your modification is correct. The one thing that changed is previously we made explicit that this was unwrapping a possible thread::AccessError specifically. But that's guaranteed by the stdlib, and I don't think the reader of this code _really_ needs to know that, so the transformation is fine. Update to remove the TODO?